src/acvp_aes.c

```diff
@@ -509,6 +511,22 @@ static ACVP_RESULT acvp_aes_mct_tc(ACVP_CTX *ctx,
     return ACVP_SUCCESS;
 }
 
+static ACVP_SYM_CIPH_TWEAK_MODE read_tw_mode(const char *str) {
```

Comment:
Nitpicky, but might we want to preface function name with acvp_?

Reply:
There's 3-4 of them and I just followed previous convention :)